Project.toml

```
@@ -1,43 +1,30 @@
 name = "LogDensityProblemsAD"
 uuid = "996a588d-648d-4e1f-a8f0-a84b347e47b1"
 authors = ["Tamás K. Papp <tkpapp@gmail.com>"]
-version = "1.9.0"
+version = "1.10.0"
```

The PR is definitely breaking (e.g., `ADgradient` is now a struct with 3 fields), so this would have to be
```suggestion
version = "2.0.0"
```

---

Is it breaking though? The internals of ADgradient are not documented, only the fact that you can call it to create "something", right?

---

It's exported and downstream code may specialize on `ADgradient`, or rather its type. And this change affects the type.

---

Ok so if ADgradient remains a `Function` and if I put the three fields in a `struct` with a different name we're good?